tests/kubeTest.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Copyright 2017 Istio Authors
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+# Local vars
+SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+K8CLI="kubectl"
+ISTIOCLI="istioctl"
+RULESDIR=$SCRIPTDIR/apps/bookinfo/rules
+EXAMPLESDIR=$SCRIPTDIR/apps/bookinfo/output
+PASS=true
+
+# Import relevant common utils
+. $SCRIPTDIR/commonUtils.sh # Must be first for print_block_echo in kubeUtils
+. $SCRIPTDIR/kubeUtils.sh
+. $SCRIPTDIR/istioUtils.sh
+
+# Setup
+deploy_istio
```

Comment:
the kubectl create would be using `kubectl create -n <namespace> -f ..`.. You would have to change the `-n default` parameter in all invocations of the manager, such as
```
+        args:
 +          - proxy
 +          - sidecar
 +          - -n
 +          - default
 +          - -s
 +          - istio-route-controller:8080
 +          - -v
 +          - "2"
```

That would allow you to test against the PR being tested and also confine the test to a specific namespace.
Although, if we are going to be triggering this test whenever there is a PR in manager/mixer, then presumably that test would already have setup the private namespace right? @sebastienvas @kyessenov ?